### PR TITLE
Python 3: Port src/conf_file

### PR DESCRIPTION
**Pull request by hahakiki2010**
All py files under src/conf_file are porting
compatible with py2 and py3, and removed autotest
dependancies

Signed-off-by: Yan Li <yannli@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/conf_file/qemu_conf/seccomp_sandbox.py [hunk 1]
```diff
@@ -30,7 +29,7 @@ def get_qemu_command_sandbox_option(vm):
 
         # Get qemu command line.
         pid = vm.get_pid()
-        res = utils.run("ps -p %s -o cmd h" % pid)
+        res = process.run("ps -p %s -o cmd h" % pid, shell=True)
 
         if res.exit_status == 0:
             match = re.search(r'-sandbox\s*(\S*)', res.stdout)
```
Comment (chunfuwen) [inline libvirt/tests/src/conf_file/qemu_conf/seccomp_sandbox.py]: res.stdout need strip()

File: libvirt/tests/src/conf_file/qemu_conf/set_process_name.py [hunk 2]
```diff
@@ -30,7 +29,7 @@ def get_qemu_command_name_option(vm):
 
         # Get qemu command line.
         pid = vm.get_pid()
-        res = utils.run("ps -p %s -o cmd h" % pid)
+        res = process.run("ps -p %s -o cmd h" % pid, shell=True)
 
         if res.exit_status == 0:
             match = re.search(r'-name\s*(\S*)', res.stdout)
```
Comment (chunfuwen) [inline libvirt/tests/src/conf_file/qemu_conf/set_process_name.py]: stdout need strip()

File: libvirt/tests/src/conf_file/qemu_conf/seccomp_sandbox.py (hunk 1 above)
Comment (hahakiki2010) [reply libvirt/tests/src/conf_file/qemu_conf/seccomp_sandbox.py]: fixup

File: libvirt/tests/src/conf_file/qemu_conf/set_process_name.py (hunk 2 above)
Comment (hahakiki2010) [reply libvirt/tests/src/conf_file/qemu_conf/set_process_name.py]: fixup

**Pull request merged**